### PR TITLE
I added Buttons Component task #7

### LINKED ISSUE #7: Storybook: Button Component
## User story

**Who:** **As a** developer
**What:** **I want to** be have a button component that can have different colors, text and width.
**Why:** **so that I can** that can be the same component in multiple pages.

## Acceptance criteria

Create a storybook component like the one we can see highlighted in pink 
 
Screenshot 2021-04-13 at 00 32 19
Screenshot 2021-04-13 at 00 52 04
Screenshot 2021-04-13 at 00 49 59

Check the wireframes here: https://www.figma.com/file/9SJRaxPSuPI3lSl8ULNyJ8/GuideIT-For-Developers?node-id=0%3A1

## Implementation details

Use **Knobs** where we can choose  **WIDTH**, **FONTS** like [Roboto](https://fonts.google.com/specimen/Roboto?preview.text=Back&preview.text_type=custom) and [Roboto+Mono](https://fonts.google.com/specimen/Roboto+Mono?preview.text=What%20is%20GuideIT%3F&preview.text_type=custom) and **COLORS** #DB643D, #535C75 and #FFFFFF

### PR DESCRIPTION
**Pull request by Poorooshka**
# Description

I want to have a button component that can have different labels, colors, text and width so it can be used all over the project along with other components.

Fixes #7 

# How to test?

type npm run storybook in your terminal and test the button component on the storybook on your browser.

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by cph-kiwi** [comment]
@Poorooshka You need to add an onClick into the .js and stories files - so that the button does something. I think the orange colour is not quite the same as the one in the wireframe. And finally, your button has a border, which the ones in the wireframe do not.

**Review by dpfernandes — COMMENTED** [review]
The orange is background: #DB643D so that seems correct @cph-kiwi - https://www.figma.com/file/9SJRaxPSuPI3lSl8ULNyJ8/GuideIT-For-Developers?node-id=1%3A1700

File: src/client/components/Buttons/Buttons.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  backgroundColor,
+}) {
+  return (
+    <div>
+      <button
+        type="submit"
+        className="buttons"
+        style={{
+          width: `${width}%`,
+          fontfamily: `${font}`,
```
Comment (soujanyapolavarapu) [inline src/client/components/Buttons/Buttons.js:18]: In React, Inline style ( fontfamily ) should be fontFamily

**Review by soujanyapolavarapu — COMMENTED** [review]
I have noticed , inline style `fontFamily` should be like this.

**Review by soujanyapolavarapu — APPROVED** [review]

File: src/client/components/Buttons/Buttons.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
```
Comment (sandhana14) [inline src/client/components/Buttons/Buttons.js:3]: I think so You forgot to import(import "./Buttons.styles.css") css file in the js

**Review by sandhana14 — CHANGES_REQUESTED** [review]

**Review by TITHI1244 — CHANGES_REQUESTED** [review]
The code looks nice. Only thing is, you need to find another way to avoid importing classnames. And also remember to delete the fonts' import in css.

**Review by ditlevHYF — APPROVED** [review]

**Pull request merged**